dronekit/__init__.py

```diff
@@ -1142,12 +1142,12 @@ def listener(vehicle, name, m):
         # All keys are strings.
         self._channels = Channels(self, 8)
 
-        @self.on_message('RC_CHANNELS_RAW')
+        @self.on_message('RC_CHANNELS')
         def listener(self, name, m):
             def set_rc(chnum, v):
                 '''Private utility for handling rc channel messages'''
                 # use port to allow ch nums greater than 8
-                self._channels._update_channel(str(m.port * 8 + chnum), v)
+                self._channels._update_channel(str(0 * 8 + chnum), v)
```

Comment:
What about simply `chnum` :-)

I'm hoping your testing will show that casting this to a string is unnecessary.

Reply:
Sure!